assets/js/script.js

```diff
@@ -1,39 +1,74 @@
-    headers: myHeaders,
-    redirect: "follow",
-  };
-  var city = $("#city").val();
-  console.log(city);
-  var apiUrl = "https://api.smartable.ai/coronavirus/news/US";
-  console.log(apiUrl);
-  // make a get request to url
-  fetch(apiUrl, requestOptions)
-    .then(function (response) {
-      // request was successful
-      if (response.ok) {
-        //call display function
-        response.json().then(function (data) {
-          displayRates(data);
-        });
-      } else {
-        alert("Error: " + response.statusText);
-      }
+
+//start of samuels portion
+$("#search").on("click",function(e) {
+    e.preventDefault()
+    var place = $("#city").val();
+      // make a get request to url
+      var api = "https://covid-19-coronavirus-statistics.p.rapidapi.com/v1/total?country=" + place;
+      console.log(api)
+      fetch(api, {
+        "method": "GET",
+        "headers": {
+            "x-rapidapi-key": "5320d39b26msh9d1c12db50862d2p1c0e8ajsn0cf78c7c1705",
+            "x-rapidapi-host": "covid-19-coronavirus-statistics.p.rapidapi.com"
+        }
     })
-    .catch(function (error) {});
+      .then(function(response) {
+        // request was successful
+        if (response.ok) {
+            //call display function
+          response.json().then(function(data) {
+              // display information
+              if (data.data.location === "Global"){
+                  errorMessage.innerHTML = "This country is not yet supported by our program, please have a look at the global stats!";
+                  errorMessage.setAttribute("class", "error");
+                countryName.innerHTML = data.data.location;
+                lastChecked.innerHTML ="updated on " + moment().format('LLL');
+                currentDeaths.innerHTML = " Current Deaths: " + data.data.deaths;
+                currentCases.innerHTML = "Current Cases: " + data.data.confirmed
+                currentRecoveries.innerHTML = "Recovered Cases: " + data.data.recovered;
+              }
+              else{
+                errorMessage.innerHTML = "";
+            countryName.innerHTML = place;
+            lastChecked.innerHTML ="updated on " + moment().format('LLL');
+            currentDeaths.innerHTML = " Current Deaths: " + data.data.deaths;
+            currentCases.innerHTML = "Current Cases: " + data.data.confirmed
+            currentRecoveries.innerHTML = "Recovered Cases: " + data.data.recovered;
+              }
+            //append information
+            
+            info.appendChild(errorMessage);
+            info.appendChild(countryName);
+            info.appendChild(lastChecked);
+            info.appendChild(currentCases);
+            info.appendChild(currentDeaths);
+            info.appendChild(currentRecoveries);
+
+
+
+          });
+        } else {
+          alert("Error: " + response.statusText);
+        }
+      })
+      .catch(function(error) {
+        alert("Unable to connect to GitHub");
+      });
```

Comment:
could you take this entire logic into its own function? There is more code that will be defined in this event listener, and we'd rather keep it clean and modular. Thanks